internal/characters/mona/cons.go

```diff
@@ -58,3 +120,82 @@ func (c *char) c4() {
 		})
 	}
 }
+
+// C6:
+// Upon entering Illusory Torrent, Mona gains a 60% increase to the DMG of her next Charged Attack per second of movement.
+// A maximum DMG Bonus of 180% can be achieved in this manner.
+// The effect lasts for no more than 8s.
+func (c *char) c6(src int) func() {
+	return func() {
+		if c.c6Src != src {
+			c.Core.Log.NewEvent(fmt.Sprintf("%v stack gain check ignored, src diff", c6Key), glog.LogCharacterEvent, c.Index).
+				Write("src", src).
+				Write("new src", c.c6Src)
+			return
+		}
+		// do nothing if not Mona
+		if c.Core.Player.Active() != c.Index {
+			return
+		}
+		// do nothing if we aren't dashing anymore
+		if c.Core.Player.CurrentState() != action.DashState {
+			return
+		}
+
+		c.c6Stacks++
+		if c.c6Stacks > 3 {
+			c.c6Stacks = 3
+		}
+		c.Core.Log.NewEvent(fmt.Sprintf("%v stack gained", c6Key), glog.LogCharacterEvent, c.Index).
+			Write("c6Stacks", c.c6Stacks)
+
+		m := make([]float64, attributes.EndStatType)
+		c.AddAttackMod(character.AttackMod{
+			Base: modifier.NewBase(c6Key, 8*60),
+			Amount: func(atk *combat.AttackEvent, t combat.Target) ([]float64, bool) {
+				if atk.Info.AttackTag != combat.AttackTagExtra {
+					return nil, false
+				}
+				m[attributes.DmgP] = 0.60 * float64(c.c6Stacks)
+				// reset stacks and remove buff on CA
+				c.c6Stacks = 0
```

Comment:
I see this and `c6CAReset()` have a similar purpose. Is there a reason they weren't consolidated?